README.md

```diff
@@ -56,7 +56,14 @@ The following attributes are available to configure your module:
 | `model_path` | string | **Required** |         | Path to **standalone** model file |
 | `label_path` | string | Optional     |         | Path to file with class labels.   |
 
+### Example configuration
 
+```json
+{
+    "model_path": "${packages.ml_model.[myMLModel]}/my_model.pt", 
+    "label_path": "${packages.ml_model.[myMLModel]}/labels.txt"
```

Comment:
I'm open to disagreement on this, but I'd remove the square brackets. We want to indicate to the user that the curly braces should literally be there in the config, and I think adding brackets you're supposed to remove confuses this.

Reply:
perfectly fine, removed the square brackets